persistence/src/test/scala/hmda/persistence/processing/SubmissionManagerSpec.scala

```diff
@@ -2,43 +2,92 @@ package hmda.persistence.processing
 
 import java.time.Instant
 
+import akka.actor.ActorRef
 import akka.testkit.TestProbe
+import akka.pattern.ask
+import akka.util.Timeout
 import com.typesafe.config.ConfigFactory
-import hmda.model.fi.{ SubmissionId, Uploaded, ValidatedWithErrors }
+import hmda.model.fi._
+import hmda.model.util.FITestData._
+import hmda.persistence.HmdaSupervisor
+import hmda.persistence.HmdaSupervisor.{ FindFilings, FindProcessingActor }
+import hmda.persistence.institutions.FilingPersistence
 import hmda.persistence.messages.CommonMessages.GetState
 import hmda.persistence.model.ActorSpec
-import hmda.persistence.processing.HmdaRawFile.AddLine
 import hmda.persistence.processing.ProcessingMessages.{ CompleteUpload, StartUpload }
-import hmda.persistence.processing.SubmissionManager._
+import hmda.persistence.institutions.FilingPersistence._
+import hmda.persistence.processing.HmdaRawFile.AddLine
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
 
 class SubmissionManagerSpec extends ActorSpec {
-  import hmda.model.util.FITestData._
+  val timeout = 2.seconds
+  implicit val akkaTimeout = Timeout(timeout)
 
   val config = ConfigFactory.load()
 
-  val submissionId = SubmissionId("0", "2017", 1)
+  val submissionId = SubmissionId("0", "testPeriod", 1)
 
-  val submissionManager = createSubmissionManager(system, submissionId)
+  val supervisor = system.actorOf(HmdaSupervisor.props(), "supervisor")
+  val fFilingPersistence = (supervisor ? FindFilings(FilingPersistence.name, submissionId.institutionId)).mapTo[ActorRef]
+  val fSubmissionManager = (supervisor ? FindProcessingActor(SubmissionManager.name, submissionId)).mapTo[ActorRef]
+  val filingPersistence: ActorRef = Await.result(fFilingPersistence, 2.seconds)
+  val submissionManager: ActorRef = Await.result(fSubmissionManager, 2.seconds)
 
   val probe = TestProbe()
 
   val lines = fiCSV.split("\n")
   val timestamp = Instant.now.toEpochMilli
 
   "A HMDA File" must {
-    "upload, parse and validate" in {
+
+    // setup: create Filing object
+    val filing = Filing(submissionId.period, submissionId.institutionId, NotStarted, filingRequired = false, 0L, 0L)
+    probe.send(filingPersistence, CreateFiling(filing))
+    probe.expectMsg(Some(filing))
+
+    "Filing status begins as 'not started'" in {
+      expectedFiling.status mustBe NotStarted
+    }
+
+    "have Filing status 'in progress' and a 'start' time after StartUpload event" in {
       probe.send(submissionManager, StartUpload)
+      Thread.sleep(200)
+
+      val filing = expectedFiling
+      filing.status mustBe InProgress
+      (filing.start == 0) mustBe false
+      filing.end mustBe 0
+    }
+
+    "upload, parse and validate" in {
```

Comment:
This test and the next one no longer have any expectations in them, so we may as well comment out the whole spec instead of individual lines.